### PR TITLE
[9.x] Add missing php extensions in composer

### PR DESCRIPTION
**Pull request by joelharkes**
Followup: https://github.com/laravel/framework/pull/45848

add missing necessary (for testing) and suggested PHP extensions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -83,6 +83,8 @@
         "illuminate/view": "self.version"
     },
     "require-dev": {
+        "ext-gmp": "*",
+        "ext-bcmath": "*",
```
Comment (driesvints) [inline composer.json:87]: Didn't we recently removed these? @timacdonald

**Comment by taylorotwell** [comment]
Waiting on @timacdonald review.

**Comment by timacdonald** [comment]
There is some additional tweaking I would like to do for this one. Will take a look on Monday. Thanks

File: src/Illuminate/Auth/composer.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Auth/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Auth/TokenGuard.php#L85

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Auth/DatabaseUserProvider.php#L76

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Auth/EloquentUserProvider.php#L83

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php#L214

File: src/Illuminate/Broadcasting/composer.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Broadcasting/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Broadcasting/Broadcasters/PusherBroadcaster.php#L57

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php#L109

File: src/Illuminate/Cache/composer.json [hunk 4]
```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Cache/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Cache/DynamoDbStore.php#L486

File: src/Illuminate/Console/composer.json [hunk 5]
```diff
@@ -15,6 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-mbstring": "*",
```
Comment (timacdonald) [inline src/Illuminate/Console/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/View/Components/Task.php#L27

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/View/Components/Task.php#L44

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Scheduling/ScheduleListCommand.php#L103

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Scheduling/ScheduleListCommand.php#L116

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Scheduling/ScheduleListCommand.php#L130-L132

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Scheduling/ScheduleListCommand.php#L151

File: src/Illuminate/Console/composer.json [hunk 6]
```diff
@@ -15,6 +15,8 @@
...
+        "ext-pcntl": "*",
```
Comment (timacdonald) [inline src/Illuminate/Console/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L79

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L95

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L79

File: src/Illuminate/Cookie/composer.json [hunk 7]
```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Cookie/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Cookie/CookieValuePrefix.php#L16

File: src/Illuminate/Database/composer.json [hunk 8]
```diff
@@ -16,7 +16,9 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-PDO": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Database/composer.json:20]: https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Query/Grammars/PostgresGrammar.php#L231

https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Query/Grammars/PostgresGrammar.php#L673

File: src/Illuminate/Database/composer.json [hunk 9]
```diff
@@ -16,7 +16,9 @@
...
+        "ext-pcntl": "*",
```
Comment (timacdonald) [inline src/Illuminate/Database/composer.json:21]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Database/Console/DatabaseInspectionCommand.php#L241

File: src/Illuminate/Database/composer.json [hunk 10]
```diff
@@ -16,7 +16,9 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-PDO": "*",
```
Comment (timacdonald) [inline src/Illuminate/Database/composer.json:19]: Search usages of `\PDO`, `\PDOStatement`, and `\PDOException`.

File: src/Illuminate/Encryption/composer.json [hunk 11]
```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Encryption/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Encryption/Encrypter.php#L257

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Encryption/Encrypter.php#L195

File: src/Illuminate/Filesystem/composer.json [hunk 12]
```diff
@@ -15,6 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-fileinfo": "*",
```
Comment (timacdonald) [inline src/Illuminate/Filesystem/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Filesystem/Filesystem.php#L452

File: src/Illuminate/Filesystem/composer.json [hunk 13]
```diff
@@ -15,6 +15,8 @@
...
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Filesystem/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Filesystem/Filesystem.php#L177

File: src/Illuminate/Http/composer.json [hunk 14]
```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Http/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Http/Concerns/InteractsWithInput.php#L341-L344

File: src/Illuminate/Pagination/composer.json [hunk 15]
```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Pagination/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Pagination/AbstractPaginator.php#L131-L134

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Pagination/PaginationState.php#L19-L27

File: src/Illuminate/Queue/composer.json [hunk 16]
```diff
@@ -15,7 +15,9 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-PDO": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/SqsQueue.php#L205-L212

File: src/Illuminate/Queue/composer.json [hunk 17]
```diff
@@ -15,7 +15,9 @@
...
+        "ext-mbstring": "*",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:20]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/Failed/DatabaseFailedJobProvider.php#L56-L65

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/Failed/DatabaseUuidFailedJobProvider.php#L56-L68

File: src/Illuminate/Queue/composer.json [hunk 18]
```diff
@@ -15,7 +15,9 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-PDO": "*",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/DatabaseQueue.php#L256-L280

File: src/Illuminate/Routing/composer.json [hunk 19]
```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
+        "ext-hash": "*",
```
Comment (timacdonald) [inline src/Illuminate/Routing/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Routing/UrlGenerator.php#L333-L350

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Routing/UrlGenerator.php#L421-L436

File: src/Illuminate/Routing/composer.json [hunk 20]
```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Routing/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Routing/UrlGenerator.php#L623-L630

File: src/Illuminate/Session/composer.json [hunk 21]
```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-ctype": "*",
```
Comment (timacdonald) [inline src/Illuminate/Session/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Session/Store.php#L649-L652

File: src/Illuminate/Session/composer.json [hunk 22]
```diff
@@ -15,7 +15,8 @@
...
+        "ext-session": "*",
```
Comment (timacdonald) [inline src/Illuminate/Session/composer.json:19]: All session handlers extend `\SessionHandlerInterface`

File: src/Illuminate/Support/composer.json [hunk 23]
```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-ctype": "*",
```
Comment (timacdonald) [inline src/Illuminate/Support/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Support/Str.php#L1053-L1068

File: src/Illuminate/Support/composer.json [hunk 24]
```diff
@@ -15,7 +15,8 @@
...
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Support/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Support/Stringable.php#L1175-L1178

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Support/Stringable.php#L260-L269

File: src/Illuminate/Testing/composer.json [hunk 25]
```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-mbstring": "*",
```
Comment (timacdonald) [inline src/Illuminate/Testing/composer.json:18]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Testing/Constraints/SeeInOrder.php#L41-L62

File: src/Illuminate/Queue/composer.json (hunk 18 above)
Comment (X-Coder264) [reply src/Illuminate/Queue/composer.json:18]: We shouldn't require the PDO extension for the entire Queue component as it's only used for the database queue. If I use the Redis queue for example I don't need PDO. IMO ext-PDO should only be in the suggest section.

File: src/Illuminate/Validation/composer.json [hunk 26]
```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
```
Comment (timacdonald) [inline src/Illuminate/Validation/composer.json:18]: See usages of `filter_var`, `FILTER_FLAG_EMAIL_UNICODE`, `FILTER_FLAG_IPV4`, `FILTER_FLAG_IPV6`, `FILTER_VALIDATE_EMAIL`, `FILTER_VALIDATE_INT`, `FILTER_VALIDATE_IP`, and `FILTER_VALIDATE_MAC`.

File: src/Illuminate/Validation/composer.json [hunk 27]
```diff
@@ -15,7 +15,8 @@
...
+        "ext-mbstring": "*",
```
Comment (timacdonald) [inline src/Illuminate/Validation/composer.json:19]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L2352-L2369

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Validation/DatabasePresenceVerifier.php#L103-L114

File: src/Illuminate/View/composer.json [hunk 28]
```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-tokenizer": "*",
```
Comment (timacdonald) [inline src/Illuminate/View/composer.json:18]: See usages of `token_get_all`, `T_CLOSE_TAG`, `T_INLINE_HTML`, `T_OPEN_TAG`, and `T_OPEN_TAG_WITH_ECHO`.

File: src/Illuminate/Queue/composer.json (hunk 18 above)
Comment (timacdonald) [reply src/Illuminate/Queue/composer.json:18]: I agree. I'm going through now and making tweaks from required to suggested 👍

Still a WIP.

File: src/Illuminate/Queue/composer.json [hunk 29]
```diff
@@ -39,6 +38,9 @@
         }
     },
     "suggest": {
+        "ext-PDO": "Required to use the database queue worker.",
+        "ext-filter": "Required to use the SQS queue worker.",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:42]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/SqsQueue.php#L205-L212

File: src/Illuminate/Queue/composer.json [hunk 30]
```diff
@@ -39,6 +38,9 @@
...
+        "ext-mbstring": "Required to use the database failed job providers.",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:43]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/Failed/DatabaseFailedJobProvider.php#L56-L65

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/Failed/DatabaseUuidFailedJobProvider.php#L56-L68

File: src/Illuminate/Queue/composer.json [hunk 31]
```diff
@@ -39,6 +38,9 @@
         }
     },
     "suggest": {
+        "ext-PDO": "Required to use the database queue worker.",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:41]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/DatabaseQueue.php#L256-L280

// cc @X-Coder264

File: src/Illuminate/Filesystem/composer.json [hunk 32]
```diff
@@ -32,7 +32,9 @@
         }
     },
     "suggest": {
+        "ext-fileinfo": "Required to use the Filesystem class.",
```
Comment (timacdonald) [inline src/Illuminate/Filesystem/composer.json:35]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Filesystem/Filesystem.php#L450-L453

File: src/Illuminate/Filesystem/composer.json [hunk 33]
```diff
@@ -32,7 +32,9 @@
...
         "ext-ftp": "Required to use the Flysystem FTP driver.",
+        "ext-hash": "Required to use the Filesystem class.",
```
Comment (timacdonald) [inline src/Illuminate/Filesystem/composer.json:37]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Filesystem/Filesystem.php#L175-L178

File: src/Illuminate/Database/composer.json [hunk 34]
```diff
@@ -36,6 +36,7 @@
         }
     },
     "suggest": {
+        "ext-filter": "Required to use the Postgres database driver.",
```
Comment (timacdonald) [inline src/Illuminate/Database/composer.json:39]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Database/Query/Grammars/PostgresGrammar.php#L225-L251

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Database/Query/Grammars/PostgresGrammar.php#L666-L677

File: src/Illuminate/Console/composer.json [hunk 35]
```diff
@@ -35,6 +36,7 @@
         }
     },
     "suggest": {
+        "ext-pcntl": "Required to use signal trapping.",
```
Comment (timacdonald) [inline src/Illuminate/Console/composer.json:39]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L77-L82

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L89-L102

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Console/Signals.php#L77-L82

File: src/Illuminate/Cache/composer.json [hunk 36]
```diff
@@ -34,6 +34,7 @@
         }
     },
     "suggest": {
+        "ext-filter": "Required to use the DynamoDb cache driver.",
```
Comment (timacdonald) [inline src/Illuminate/Cache/composer.json:37]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Cache/DynamoDbStore.php#L484-L495

File: src/Illuminate/Broadcasting/composer.json [hunk 37]
```diff
@@ -35,6 +34,7 @@
         }
     },
     "suggest": {
+        "ext-hash": "Required to use the Ably and Pusher broadcast drivers.",
```
Comment (timacdonald) [inline src/Illuminate/Broadcasting/composer.json:37]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Broadcasting/Broadcasters/PusherBroadcaster.php#L43-L65

https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php#L107-L114

File: src/Illuminate/Database/composer.json [hunk 38]
```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-pdo": "*",
```
Comment (timacdonald) [inline src/Illuminate/Database/composer.json:19]: Search usages of `\PDO`, `\PDOStatement`, and `\PDOException`.

File: src/Illuminate/Queue/composer.json [hunk 39]
```diff
@@ -39,6 +38,9 @@
         }
     },
     "suggest": {
+        "ext-pdo": "Required to use the database queue worker.",
```
Comment (timacdonald) [inline src/Illuminate/Queue/composer.json:41]: https://github.com/laravel/framework/blob/d3a4cd268bd716838482125f36ccf6bf8d51088a/src/Illuminate/Queue/DatabaseQueue.php#L256-L280

// cc @X-Coder264

File: composer.json (hunk 1 above)
Comment (timacdonald) [reply composer.json:87]: We were still referencing bcmath deps in some random tests. I've refactored those away now.

File: composer.json [hunk 40]
```diff
@@ -83,6 +88,7 @@
         "illuminate/view": "self.version"
     },
     "require-dev": {
+        "ext-gmp": "*",
```
Comment (timacdonald) [inline composer.json:91]: I think we still need this in `require-dev`.

See this PR: https://github.com/laravel/framework/pull/43959

Seems best to leave it as GMP objects have some quirks that this addresses and I can't replicate with standard classes.

File: composer.json [hunk 41]
```diff
@@ -16,8 +16,13 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-ctype": "*",
```
Comment (timacdonald) [inline composer.json:19]: Required in `illuminate/support` and `illuminate\session`.

File: composer.json [hunk 42]
```diff
@@ -16,8 +16,13 @@
...
+        "ext-filter": "*",
```
Comment (timacdonald) [inline composer.json:20]: Required in:

- `illuminate/routing`
- `illuminate/validation`
- `illuminate/support`
- `illuminate/http`
- `illuminate/pagination`

Optional in:

- `illuminate/queue`
- `illuminate/cache`
- `illuminate/database`

File: composer.json [hunk 43]
```diff
@@ -16,8 +16,13 @@
...
+        "ext-hash": "*",
```
Comment (timacdonald) [inline composer.json:21]: Required in:

- `illuminate/auth`
- `illuminate/routing`
- `illuminate/cookie`
- `illuminate/encryption`

Optional in:

- `illuminate/filesystem`
- `illuminate/broadcasting`

File: composer.json [hunk 44]
```diff
@@ -16,8 +16,13 @@
...
         "ext-mbstring": "*",
         "ext-openssl": "*",
+        "ext-session": "*",
```
Comment (timacdonald) [inline composer.json:24]: Required in:

- `illuminate/session`

File: composer.json [hunk 45]
```diff
@@ -16,8 +16,13 @@
...
+        "ext-tokenizer": "*",
```
Comment (timacdonald) [inline composer.json:25]: Required in:

- `illuminate/view`

File: src/Illuminate/Cache/composer.json [hunk 46]
```diff
@@ -34,6 +34,8 @@
         }
     },
     "suggest": {
+        "ext-apcu": "Required to use the APC cache driver.",
```
Comment (timacdonald) [inline src/Illuminate/Cache/composer.json:37]: See the `ApcWrapper` class.

**Pull request merged**